fbfmaproom/fbfmaproom.py

```diff
@@ -47,7 +47,7 @@
 SERVER.register_error_handler(ClientSideError, pingrid.client_side_error)
 
 month_abbrev = ['jan', 'feb', 'mar', 'apr', 'may', 'jun', 'jul', 'aug', 'sep', 'oct', 'nov', 'dec']
-abbrev_to_month0 = dict((abbrev, month0) for month0, abbrev in enumerate(month_abbrev))
+abbrev_to_month0: Dict[str, int]  = dict((abbrev, month0) for month0, abbrev in enumerate(month_abbrev))
```

Comment:
@jpngfile if you're motivated to set up type checking in Github Actions, I would enthusiastically support that.

Reply:
(Before embarking on that, talk to me. Topics to discuss: which type checker, managing the pixi/conda environment.)